zuul-core/src/main/java/com/netflix/netty/common/throttle/RejectionUtils.java

```diff
@@ -162,6 +178,14 @@ public static void handleRejection(
         }
     }
 
+
+    public static void reject(
```

Comment:
Later linters nag on missing javadoc for public methods,  consider copying some of the existing docs for the param names.